index.html

```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
         </div>
        </section>
     </section>
-
+    <section class="first-project">
+        <div class="snapshot"></div>
+        <div class="content-project1">
+            <h2>W3 project</h2>
+            <div class="ali">
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>An open source code for every coding language. W3 school is one the most usefull websites.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
```

Comment:
- [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.